R/get-pit-histogram.R

```diff
@@ -24,12 +26,22 @@
 #' @inheritParams pit_histogram_sample
 #' @return A data.table with density values for each bin in the PIT histogram.
 #' @examples
+#' library("ggplot2")
+#'
 #' example <- as_forecast_sample(example_sample_continuous)
 #' result <- get_pit_histogram(example, by = "model")
+#' ggplot(result,  aes(x = mid, y = density)) +
```

Comment:
we might wrap those in `not run`, but not sure it's actually necessary

Reply:
happy to add but given that `ggplot2` is imported I don't think this will affect checks etc.

Reply:
I was more thinking about build times and CRAN, but maybe let's not worry about that now